### PR TITLE
Fix #474 Vertical Axes Title Font Bug

### LINKED ISSUE #474: Vertical Axes Title Font Bug
Hi!

When I change the font to "Times New Roman" or "Arial" the title of vertical axes disappears

```c#
            var model = new PlotModel ();
            model.Axes.Add(new LinearAxis
            {
                Title = "X_Axe",
                Position = AxisPosition.Bottom,
                MajorGridlineStyle = LineStyle.Solid,
                TitleFont = "Times New Roman"
            });
            model.Axes.Add(new LinearAxis
            {
                Title = "Y_Axe",
                Position = AxisPosition.Left,
                MajorGridlineStyle = LineStyle.Solid,
                TitleFont = "Times New Roman"
            });
```
plotdefault

### PR DESCRIPTION
**Pull request by Kaplas80**
Fixes #474 Vertical Axes Title Font Bug.

### Checklist

- [X] I have included examples or tests
- [X] I have updated the change log
- [X] I am listed in the CONTRIBUTORS file
- [X] I have cleaned up the commit history (use rebase and squash)

### Changes proposed in this pull request:

-
-
-

@oxyplot/admins

### CODE REVIEW

File: Source/OxyPlot.WindowsForms/GraphicsRenderContext.cs [hunk 1]
```diff
@@ -311,8 +311,7 @@ public override void DrawRectangle(OxyRect rect, OxyColor fill, OxyColor stroke,
 
                 this.g.TranslateTransform(dx, dy);
 
-                var layoutRectangle = new RectangleF(0, 0, size.Width + 0.1f, size.Height + 0.1f);
-                this.g.DrawString(text, font, fill.ToBrush(), layoutRectangle, this.stringFormat);
+                this.g.DrawString(text, font, fill.ToBrush(), 0.0f, 0.0f, this.stringFormat);
```
Comment (objorke) [inline Source/OxyPlot.WindowsForms/GraphicsRenderContext.cs]: no more support for clipped text? We want to keep that feature - see the example with a long title that is clipped

File: Source/OxyPlot.WindowsForms/GraphicsRenderContext.cs [hunk 2]
```diff
@@ -311,8 +311,7 @@ public override void DrawRectangle(OxyRect rect, OxyColor fill, OxyColor stroke,
 
                 this.g.TranslateTransform(dx, dy);
 
-                var layoutRectangle = new RectangleF(0, 0, size.Width + 0.1f, size.Height + 0.1f);
```
Comment (objorke) [inline Source/OxyPlot.WindowsForms/GraphicsRenderContext.cs]: maybe increasing 0.1f will help?

**Comment by Kaplas80** [comment]
My tests worked if I changed 0.1f to 0.5f, but I don't know if it is enough for all fonts.

**Comment by tibel** [comment]
I have fixed the appveyor build.
Please rebase, so we can verify that all tests pass.

**Pull request closed without merging**